src/transaction.ts

```diff
@@ -0,0 +1,234 @@
+import { Psbt, networks } from 'liquidjs-lib';
+import { estimateTxSize, UtxoInterface } from './wallet';
+
+export interface OutputInterface {
+  value: number;
+  asset: string;
+  script: string;
+}
+
+export interface CoinSelectionResult {
+  selectedUtxos: UtxoInterface[];
+  changeOutputs: OutputInterface[];
+}
+
+// define a type using to implement changeScript strategy
+type ChangeScriptFromAssetGetter = (asset: string) => string | undefined;
+
+/**
+ * buildTx selects utxos among unspents to fill outputs' requirements,
+ * then it adds the associated inputs and outputs to the tx.
+ * finally it returns the new pset base64 encoded.
+ * @param psetBase64
+ * @param unspents
+ * @param outputs
+ * @param changeScriptByAsset
+ * @param addFee if true, estimate fee and add feeOutput to transaction (default = false)
+ * @param satsPerByte used for fee estimation (default = 0.1)
+ * @param network used for fee output (default = regtest)
+ */
+export function buildTx(
+  psetBase64: string,
+  unspents: UtxoInterface[],
+  outputs: OutputInterface[],
```

Comment:
woudl call `recipients` to make it more high level for end-users and not conflate with tx output